src/app/screens/Home/index.tsx

```diff
@@ -94,6 +100,33 @@ function Home() {
     }
   }
 
+  async function onTabChangeHandler(index: number) {
+    if (index === 1) {
+      await loadInvoices();
+    }
+  }
+
+  async function loadInvoices() {
+    setLoadingInvoices(true);
+    const result = await api.getInvoices();
+
+    const invoices: Transaction[] = result.invoices
+      .filter((invoice) => invoice.settled)
+      .map((invoice) => ({
+        ...invoice,
+        title: invoice.memo,
+        date: dayjs(invoice.settleDate).fromNow(),
```

Comment:
somehow I would like that this data would be coming in a usable format from the action. that less filtering/transformation is done in the "frontend".
This would make it more re-usable.

maybe something to refactor in the future?

Reply:
Yeah, I was following the payment-example:  
https://github.com/getAlby/lightning-browser-extension/blob/feat/389_tx-lists-incoming-invoices/src/app/screens/Home/index.tsx#L238
Can be something for the future I believe. I.e. as soon as we show payments/invoices somewhere else and also need to use the i.e. `dayjs`-function

Reply:
Should be refactored in a TODO. Once this and further TS-switches for allowances are merged.

Reply:
https://github.com/getAlby/lightning-browser-extension/issues/1158